### DISCUSSION TITLE
Add to album based on location when imported?

### DISCUSSION THREAD

**Discussion by JamesSwift** [photoprism/photoprism#1076] [Q&A] [answered]
When I imported my library, some photos were in folders with an album name. Now that everything has been imported, I'd like to recreate these albums within PP. When I open the info view for one of these photos, I can still see the original file path (including tha folder/album name), but there seems to be no way to search and select a group of photos based on this information. Any suggestions?

To be clear, I "imported", rather moving my photos in into the "originals" directory. So, the originals directory now only contains date-based folders. When I go to the "Folders" menu item, I just see months and years.

**Comment by alexisLefebvre** [comment, marked as answer]
Try the `original` filter with something like `original:"*my_folder/*"`

Doc: https://docs.photoprism.org/user-guide/organize/search/#search-filter-list

**Reply by JamesSwift** [reply]
Excellent! Thank you. I searched but missed that bit in the docs. 

Is there any way to convince the "Folders" or "Originals" view to let you browse in this way?

**Reply by lastzero** [reply]
No, the current folders are shown. We may later add dynamic search filters for this.
